metagraph/tests/graph/all/test_dbg_basic.cpp

```diff
@@ -65,7 +61,7 @@ TYPED_TEST(DeBruijnGraphTest, SerializeEmpty) {
     EXPECT_TRUE(check_graph_nodes(graph));
 }
 
-TYPED_TEST(DeBruijnGraphTest, Serialize) {
```

Comment:
So we can either add load/serialize to `DBGSuccinctCachedView`, or to the parent `DBGWrapper`. My original plan was to add it to the latter (since I wanted to leave it up to the graph implementations to decide if they can be loaded or serialized), but you objected to that:
https://github.com/ratschlab/metagraph/pull/340#discussion_r676116419

Reply:
You don't need to keep another non-const pointer to have load/serialize. So this doesn't contradict my last comment.

Reply:
Sure, this can be done with a `const_cast` if you trust the user to know what they're doing.

Reply:
anyway, I added this